File: 01_ProgrammingBasics/_06_NestedLoopsMoreExercises/_07_SafePasswordsGenerator.java
package _01_ProgrammingBasics._06_NestedLoopsMoreExercises;

import java.util.Scanner;

public class _07_SafePasswordsGenerator {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        int a = Integer.parseInt(scan.nextLine());
        int b = Integer.parseInt(scan.nextLine());
        int max = Integer.parseInt(scan.nextLine());

        // Add variables
        char A = '#';
        char B = '@';
        int count = 0;

        // Print generated passwords
        forOuter:
        for (int i = 1; i <= a; i++) {
            for (int j = 1; j <= b; j++) {
                if (count == max) {
                    break forOuter;
                }
                
                if (A == 56) {
                    A = '#';
                }
                
                if ( B == 97) {
                    B = '@';
                }

                System.out.printf("%s%s%d%d%s%s|", A, B, i, j, B++, A++);
                count++;
            }
        }

        scan.close();
    }
}

File: 23_C#Basics/_06_NestedLoopsMoreExercises/_07_SafePasswordsGenerator/_07_SafePasswordsGenerator.cs
using System;

public class _07_SafePasswordsGenerator
{
    public static void Main(string[] args)
    {
        // Read input values
        int a = int.Parse(Console.ReadLine());
        int b = int.Parse(Console.ReadLine());
        int max = int.Parse(Console.ReadLine());

        // Add variables
        char A = '#';
        char B = '@';
        int count = 0;
        bool breakOuter = false;

        for (int i = 1; i <= a && !breakOuter; i++)
        {
            for (int j = 1; j <= b; j++)
            {
                if (count == max)
                {
                    breakOuter = true;
                    break; // Break inner loop
                }

                if (A == 56)
                {
                    A = '#';
                }

                if (B == 97)
                {
                    B = '@';
                }

                Console.Write($"{A}{B}{i}{j}{B++}{A++}|");
                count++;
            }
        }
    }
}
